modules/ding_place2book/ding_place2book.module

```diff
@@ -584,10 +532,9 @@ function ding_place2book_event_op($op, $node, $place2book_id = NULL) {
 
       // Catch update error by checking the return code.
       if ($req_result->status_message != 'Accepted') {
-        // @todo: This error handling is not appropriate for normal editors.
-        // Replace with drupal_set_message + watchdog.
```

Comment:
This `todo` still applies. The message displayed to the editor is not very friendly/understandable.